src/test/java/reposense/commits/CommitInfoAnalyzerTest.java

```diff
@@ -135,18 +135,18 @@ public void analyzeCommits_emailWithAdditionOperator_success() {
 
     @Test
     public void analyzeCommits_duplicateAuthorsDuplicateCommits_success() throws Exception {
```

Comment:
Why is the testing for empty author tag being done inside `duplicate`? Either create a new test or modify this test name.

Reply:
Hmm, this test case isn't testing for the empty author tag. The code in this test was updated to match the latest commit information, because a new commit was added to add more lines to the annotationTest.java file. Thus, the `LATEST_COMMIT_HASH` has to be updated as well as this test.

Reply:
Oh so got updated with the fake author name, got it. Can add a test for testing the empty author tag.

Reply:
This test class is mainly testing the analysis of commits, perhaps you were referring to `AnnotatorAnalyzerTest`? The two test cases there (`analyzeAnnotation_authorNamePresentInConfig_overrideAuthorship` and `analyzeAnnotation_authorNameNotInConfig_disownCode`) covers the empty author tag, which is based on the updated `annotationTest.java` file where I added some empty author tags.

Reply:
@Tejas2805 Ping

Reply:
Sorry for the delayed response. Thanks for the ping.